raspberry/main.py

```diff
@@ -293,7 +295,7 @@ def read_ToF_run_kf_and_LQR():
 
         # Reading the tof altitude invokes the automatic update from the sensor,
         # no need to read the sensor explicitly
-        temp = tof.altitude
+        temp = tof.distance
```

Comment:
`temp` is not a good variable name

Reply:
It's from the old TOF sensor's part, should we replace it with another name?

Reply:
Old code isn't sacred